container/src/constants/communication.ts

```diff
@@ -1,176 +1,413 @@
 
   /**
    * Event fired when the micro frontend requests to set the view group data.
+   * It's dispatched with custom data.
+   * @type {Object.<string, string>}
+   * @returns {void}
+   *
+   * <br><br>
    */
   export const SET_VIEW_GROUP_DATA_REQUEST = 'set-viewgroup-data-request';
 
   /**
-   * Event fired when the micro frontend requests to set the document title.
+   * Event left due to historical reasons - do not use.
    * @deprecated
+   * @ignore
+   *
+   * <br><br>
    */
   export const SET_DOCUMENT_TITLE_REQUEST = 'set-document-title-request';
 
   /**
-   * Event fired when the micro frontend requests to open the user settings.
+   * Event left due to historical reasons - do not use.
+   * @deprecated
+   * @ignore
+   *
+   * <br><br>
    */
   export const OPEN_USER_SETTINGS_REQUEST = 'open-user-settings-request';
 
   /**
-   * Event fired when the micro frontend requests to close the user settings.
+   * Event left due to historical reasons - do not use.
+   * @deprecated
+   * @ignore
+   *
+   * <br><br>
    */
   export const CLOSE_USER_SETTINGS_REQUEST = 'close-user-settings-request';
 
   /**
-   * Event fired when the micro frontend requests to collapse left side navigation.
+   * Event left due to historical reasons - do not use.
+   * @deprecated
+   * @ignore
+   *
+   * <br><br>
    */
   export const COLLAPSE_LEFT_NAV_REQUEST = 'collapse-leftnav-request';
 
   /**
-   * Event fired when the micro frontend requests to update the top navigation.
+   * Event left due to historical reasons - do not use.
+   * @deprecated
+   * @ignore
+   *
+   * <br><br>
    */
   export const UPDATE_TOP_NAVIGATION_REQUEST = 'update-top-navigation-request';
 
   /**
-   * Event fired when the micro frontend requests to check if the path exists.
-   * @deprecated use CHECK_PATH_EXISTS_REQUEST
+   * Event left due to historical reasons - use 'CHECK_PATH_EXISTS_REQUEST' instead.
+   * @deprecated
+   * @ignore
+   *
+   * <br><br>
    */
   export const PATH_EXISTS_REQUEST = 'path-exists-request';
 
   /**
    * Event fired when the micro frontend requests to navigate back.
+   * It's dispatched with custom data.
+   * @type {unknown}
+   * @returns {void}
+   *
+   * <br><br>
    */
   export const GO_BACK_REQUEST = 'go-back-request';
 
   /**
-   * Event fired when the micro frontend requests to check if there are any preserved views.
+   * Event left due to historical reasons - do not use.
+   * @deprecated
+   * @ignore
+   *
+   * <br><br>
    */
   export const HAS_BACK_REQUEST = 'has-back-request';
 
   /**
    * Event fired when the micro frontend requests to display the backdrop.
+   * It's dispatched with custom data.
+   * @type {Object}
+   * @returns {void}
+   *
+   * <br><br>
    */
   export const ADD_BACKDROP_REQUEST = 'add-backdrop-request';
 
   /**
    * Event fired when the micro frontend requests to remove the backdrop.
+   * It's dispatched with custom data.
+   * @type {Object}
```

Comment:
Since we don't offer any payload which can be used we should not mention it in our docu.